opentracing-noop/src/main/java/io/opentracing/noop/NoopTracerFactory.java

```diff
@@ -13,12 +13,18 @@
  */
 package io.opentracing.noop;
 
+import io.opentracing.ActiveSpanSource;
+
 public final class NoopTracerFactory {
     
     public static NoopTracer create() {
         return NoopTracerImpl.INSTANCE;
     }
 
+    public static NoopTracer create(ActiveSpanSource spanSource) {
+        return new NoopTracerImpl(spanSource);
```

Comment:
at least now it makes sense don't to have  `NoopTracerImpl.INSTANCE` public. Before I was curious why there is a factory with only one method which returns ` NoopTracerImpl.INSTANCE`